workos/utils/_base_http_client.py

```diff
@@ -32,8 +33,8 @@ class PreparedRequest(TypedDict):
     method: str
     url: str
     headers: httpx.Headers
-    params: NotRequired[Union[Dict, None]]
-    json: NotRequired[Union[Dict, None]]
+    params: NotRequired[Union[Mapping, None]]
```

Comment:
Oh. I just did the same thing in my PR. That's fine, git diffing should be good about any conflicts

Reply:
Yeah, I figured it should be okay. Just wanted to get my tests passing.